server/base/src/main/java/org/apache/accumulo/server/init/Initialize.java

```diff
@@ -762,6 +764,17 @@ public void execute(final String[] args) {
       if (opts.resetSecurity) {
         AccumuloServerContext context = new AccumuloServerContext(new ServerConfigurationFactory(HdfsZooInstance.getInstance()));
         if (isInitialized(fs)) {
+          if (!opts.forceResetSecurity) {
+            String secret = acuConf.get(Property.INSTANCE_SECRET);
+            ConsoleReader c = getConsoleReader();
+            String userEnteredSecret = c.readLine("WARNING: This will remove all users from Accumulo! If you wish to proceed enter the instance secret: ", '*');
```

Comment:
> This will remove all users from Accumulo

Really, this is removing all ZooKeeper data. The "user database" just happens to be in ZooKeeper. Maybe there's a better way to word this?